packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/Filters/index.ts

```diff
@@ -0,0 +1 @@
+export { Filters } from './Filters';
```

Comment:
Personally, I don't like imports like`*/Filters/Filters.tsx`. It is one of the reasons I use re-export.
Should I remove it?

Reply:
Let's rename it to `ApprovalRequestsFilter` then

Reply:
renamed

Reply:
@saladNights, you know, there's a practical point here. In the Next.js support branch, due to the change of distribution delivery method (multiple `/dist/**/**` files instead of one `dist/index.js`) such exports from `index.ts` were led to a [circular dependency](https://github.com/team-monite/monite-sdk/pull/129/commits/1b53b2dded0a7c748ce76779bbaf8431ba153d8f#diff-a0a8156eff8e2b89a7dfc800ca62b7557547edd6472718313804d1f0149813f1) problem.
But in any case, I don't have a preference on this, as I added [a special Rollup plugin](https://github.com/team-monite/monite-sdk/pull/129/commits/f541b6e1b91a529672f59e8301710ef041505ea4) that will fail the build when circular dependencies added. So, we are safe.

Reply:
got it, re-export removed